src/main/resources/yamlschema/README.md

```diff
@@ -1,9 +1,19 @@
 # Print Provider Schema
-# v0.0.5
+# v1.0.0
 This folder contains the Print Provider schema files.
 
 The schema is defined in `yamlschema`. Each type is defined in its own file.
 
+## Change History
```

Comment:
Helpful summary 👍 
Is this only for specs that are shared externally with third parties?
Or something we may start doing with internal specs to?

Reply:
Should this be `Print Request Schema Change History`?  (Do we need section for the `Print Response` and `Print Responses` specs?

Reply:
I don't mind us putting a change history on our internal spec - totally happy for that if we feel it has value?

This is an external spec, and given that changes are negotiated and agreed and the spec versioned, I felt it was an additional layer of documentation/professionalism to document the change history. Sure, we have the git history, but I felt this was easier and quicker to reference.

Re: should there be 2 change histories? One for `Print Request` and one for `Print Response`? I don't think so. The schema should be considered one schema for the EROP -> Print Provider integration. Even though it's defined in separate files and contains 2 logical things (requests and responses), overall it's one schema. This is why all the yaml files in the schema were up'versioned, even though individually they had nothing to do with the change being made here.